### PR TITLE
Feat/calling lua ts

### PR DESCRIPTION
**Pull request by itschip**

### CODE REVIEW

**Review by jfrader — APPROVED** [review]
Exceptional work dude!

Just small things here and there but quick things.

File: phone/src/apps/dialer/hooks/useDailService.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { useSetRecoilState } from 'recoil'
+import { useNuiEvent } from '../../../os/nui-events/hooks/useNuiEvent';
+import { dailerState } from './state';
+import { useDailHistory } from './useDailHistory';
+
+export const useDailService = () => {
+  const setHistory = useSetRecoilState(dailerState.history);
+  useNuiEvent('DAILER', 'setHistory', setHistory);
```
[the remark this answers is not in the document]
Comment (itschip) [reply phone/src/apps/dialer/hooks/useDailService.ts]: Fixed

File: phone/src/apps/dialer/hooks/useDailHistory.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { useRecoilValue } from 'recoil';
+import { dailerState } from './state';
+
+export const useDailHistory = () => {
+  const history = useRecoilValue(dailerState.history);
```
[the remark this answers is not in the document]
Comment (itschip) [reply phone/src/apps/dialer/hooks/useDailHistory.ts]: Fxied

File: phone/src/apps/dialer/hooks/state.ts [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import { atom } from 'recoil';
+
+export const dailerState = {
```
[the remark this answers is not in the document]
Comment (itschip) [reply phone/src/apps/dialer/hooks/state.ts]: Fixed

File: phone/src/apps/dialer/hooks/state.ts [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+  history: atom({
+    key: 'dailerHistory',
```
[the remark this answers is not in the document]
Comment (itschip) [reply phone/src/apps/dialer/hooks/state.ts]: Fixed

File: phone/src/apps/dialer/hooks/useDailHistory.ts [hunk 5]
```diff
@@ -0,0 +1,7 @@
+import { useRecoilValue } from 'recoil';
+import { dailerState } from './state';
+
+export const useDailHistory = () => {
```
[the remark this answers is not in the document]
Comment (itschip) [reply phone/src/apps/dialer/hooks/useDailHistory.ts]: Fixed

File: resources/client/cl_call.ts [hunk 6]
```diff
@@ -34,10 +34,16 @@ on(`__cfx_nui:${events.PHONE_ACCEPT_CALL}`, (data: any) => {
   emitNet(events.PHONE_ACCEPT_CALL, data.transmitterNumber);
 });
 
+<<<<<<< HEAD
```
[the remark this answers is not in the document]
Comment (itschip) [reply resources/client/cl_call.ts]: Fixed

File: resources/server/sv_call.ts [hunk 7]
```diff
@@ -44,10 +70,17 @@ onNet(events.PHONE_INITIALIZE_CALL, async (phoneNumber: string) => {
   calls.set(transmitterNumber, {
     transmitter: transmitterNumber,
     transmitterSource: _source,
+    transmitterIdentifier: xTransmitter.getIdentifier(),
     receiver: receiverNumber,
     receiverSource: xReceiver.source,
+<<<<<<< HEAD
```
[the remark this answers is not in the document]
Comment (itschip) [reply resources/server/sv_call.ts]: Fixed

File: phone/src/Phone.tsx [hunk 8]
```diff
@@ -258,6 +304,10 @@ function Phone() {
   useMessagesService();
   useNotesService();
   usePhotoService();
+  useCallService();
+  useDailService();
```
[the remark this answers is not in the document]
Comment (itschip) [reply phone/src/Phone.tsx]: Fixed

File: phone/src/Phone.tsx [hunk 9]
```diff
@@ -52,6 +55,49 @@ if (process.env.NODE_ENV === 'development') {
     );
   }, 1000);
 
+  setTimeout(() => {
+    window.dispatchEvent(
+      new MessageEvent('message', {
+        data: {
+          app: 'DAILER',
```
[the remark this answers is not in the document]
Comment (itschip) [reply phone/src/Phone.tsx:62]: Fixed

**Pull request merged**